### DISCUSSION TITLE
When I use skywalk-client-js on the front end ,some ajax error and Load failed appears in logs.

### DISCUSSION THREAD

**Discussion by baixjian** [apache/skywalking#10700] [Q&A]
When I use skywalk-client-js on the front end ,some ajax error and Load failed appears in logs.
<img width="1878" alt="企业微信截图_a4b68a42-e550-46f3-a62d-602cca86c386" src="https://user-images.githubusercontent.com/109275788/232723215-ad2fa8a6-4927-4188-ab6f-201c41952f16.png">

this is my config
```
export default function runSwClient() {
  ClientMonitor.register({
    service: 'xx',
    pagePath: location.href,
    serviceVersion: 'v1.0.0',
    jsErrors: true,
    apiErrors: true,
    resourceErrors: true,
    vue: Vue,
    autoTracePerf: true,
    useFmp: true,
    enableSPA: true,
    traceSDKInternal: false,
    detailMode: true,
    noTraceOrigins: [],
    traceTimeInterval: 10000,
  })
`
someone who can tell me the reason why these logs appears, 

**thanks**

**Comment by wu-sheng** [comment]
These are the browser-side error logs. The client-js agent collects it automatically.

**Reply by baixjian** [reply]
Are some **promises** without a message because they are not caught ？
[unhandledrejection](https://github1s.com/apache/skywalking-client-js/blob/v0.9.0/src/errors/promise.ts#L46)
Should I need to throw an exception in the catch myself

**Reply by wu-sheng** [reply]
@Fine0830 Could you explain a little more? I am not familiar with this.

**Comment by Fine0830** [comment]
It seems the promise errors have no messages and stack. Not all errors have the message and the stack.

**Comment by baixjian** [comment]
Thank you for your reply.  It's a pity we can't get the message.  Can I filter these logs or get more information about this promise.

**Comment by Fine0830** [comment]
Refer to https://github.com/apache/skywalking-client-js#parameters, you can set the `jsError` to false.
